### PR TITLE
Added all actions inside controller file and added index,show,new,edi…

### PR DESCRIPTION
**Pull request by Invisiblebrush7**
I added all the actions inside the bundles controller file with the necessary code for every action. I added a small validation to avoid trying to find a bundle that doesn't exist too

I created the views that were necessary and added a small h1.

### CODE REVIEW

File: app/controllers/bundles_controller.rb [hunk 1]
```diff
@@ -1,2 +1,53 @@
...
+    if BundlesOffer.exists?(params[:id])
+      @bundle = BundlesOffer.find(params[:id])
+    else
+      redirect_to bundles_url
+    end
+  end
+
+  def update
+    @bundle = BundlesOffer.find(params[:id])
+    @bundle.update(tasks_params)
```
Comment (thearavindhs) [inline app/controllers/bundles_controller.rb:37]: Please review if this is correct - @bundle.update(tasks_params)

**Review by thearavindhs — CHANGES_REQUESTED** [review]
Please review if this is correct - @bundle.update(tasks_params)

File: app/controllers/bundles_controller.rb (hunk 1 above)
Comment (thearavindhs) [inline app/controllers/bundles_controller.rb]: Please review  @bundle.update(tasks_params)

**Review by thearavindhs — COMMENTED** [review]
Please review @bundle.update(tasks_params)

File: app/controllers/bundles_controller.rb (hunk 1 above)
Comment (Invisiblebrush7) [reply app/controllers/bundles_controller.rb]: Oops, I need to change that

**Review by Invisiblebrush7 — COMMENTED** [review]
Changed that little error I made 👍

File: app/controllers/bundles_controller.rb (hunk 1 above)
Comment (thearavindhs) [reply app/controllers/bundles_controller.rb]: cool,  thanks

**Pull request merged**